package.json

```diff
@@ -50,6 +51,7 @@
     "d3-scale": "1.0.7",
     "d3-selection": "1.2.0",
     "d3-shape": "1.2.0",
+    "enzyme-to-json": "^3.3.4",
```

Comment:
Shouldn't this be in `devDependencies`? https://github.com/adriantoine/enzyme-to-json#install

Reply:
🤦‍♂️